src/scss/_drop-down.scss

```diff
@@ -5,19 +5,37 @@
 	.o-header-services {
 		li[data-o-header-services-level] {
 			position: relative;
-		}
+			min-height: 40px;
 
-		li[data-o-header-services-level] > a {
-			padding-right: oTypographySpacingSize(10);
+			> a {
+				display: inline-block;
+			}
 
-			&:after {
-				@include oIconsGetIcon('arrow-down', _oHeaderServicesGet('nav-text'), 24px);
-				content: '';
+			button {
```

Comment:
Bit unsure about styling `button` and not making it an element in BEM. If you're set on the element selector should we qualify it further with a direct descendent selector?